app/src/main/java/no/usn/mob3000/ui/screens/auth/CreateUserScreen.kt

```diff
@@ -69,6 +70,7 @@ fun CreateUserScreen(
                     value = mail,
                     onValueChange = { mail = it },
                     label = { Text(stringResource(R.string.auth_login_email)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).